invoices/update.go

```diff
@@ -59,6 +63,10 @@ const (
 	// ResultMppInProgress is returned when we are busy receiving a mpp payment.
 	ResultMppInProgress
 
+	// ResultMPPTimeout is returned when an invoice paid with multiple partial
+	// payments times out before it is fully paid.
+	ResultMPPTimeout
```

Comment:
`MPP` all upper not consistent with `ResultMppInProgress`

Reply:
Agreed on matching case in this enum. We already have `MPP` and `Mpp` in the codebase, maybe we should pick one?

Reply:
I don't know if `Mpp` is also in the Oxford 2020. I would prefer that, but it is your call.